### ISSUE TITLE
Unable to unlist my nuget package People.API.

### ISSUE THREAD

**Issue by nick301** [NuGet/NuGetGallery#4369]
Dear Team,

Please help me in unlisting my package People.API from the nuget. I don't want this package to appear in the public Nuget repository. 

Regards,
Nishant Kumar

**Comment by skofman1** [comment]
Hi Nishant, you can unlist your package using the package page: https://www.nuget.org/packages/People.Api/
When logged in, you will see on the left "Delete package" link, that will take you to the unlist option.

**Comment by nick301** [comment]
Hi,
I tried following the suggestion provided by you, but unfortunately, **I am repeatedly getting the same error**, due to which I am unable to unlist/delete my package.

"Sorry, an error occurred while processing your request. Error: Oh no, we broke something!"

Attached the error screenshot below.
![nugeterror](https://user-images.githubusercontent.com/30075842/28102647-37369d52-66ee-11e7-9daa-70d3f327b649.png)

Regards,
Nishant Kumar
